### PR TITLE
Add API3 docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the API3 docs initially submitted in https://github.com/PureStake/moonbeam-docs/pull/599

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jixa
- [x] If this page requires a disclaimer, I have added one

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/integrations/oracles/api3.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+## Using dAPIs - API3 Datafeeds {: #dapis }
+
+[dAPIs](https://docs.api3.org/explore/dapis/what-are-dapis.html){target=_blank} are continuously updated streams of off-chain data, such as the latest cryptocurrency, stock, and commodity prices. They can power various decentralized applications such as DeFi lending, synthetic assets, stablecoins, derivatives, NFTs, and more.
+
+The data feeds are continuously updated by [first-party oracles](https://docs.api3.org/explore/introduction/first-party.html){target=_blank} using signed data. DApp owners can read the on-chain value of any dAPI in realtime.
+
+Due to being composed of first-party data feeds, dAPIs offer security, transparency, cost-efficiency, and scalability in a turn-key package.
+
+![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-3.png)
```
Comment (albertov19) [inline builders/integrations/oracles/api3.md:140]: Transparent bg

File: builders/integrations/oracles/api3.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+With self-funded dAPIs, you can fund the dAPI with your own funds. The amount of gas you supply will determine how long your dAPI will be available for use. If you run out of gas, you can fund the dAPI again to keep it available for use.
+
+The process for accessing self-funded data feeds is as follows:
+
+1. Explore the API3 Market and select a dAPI
+2. Fund a sponsor wallet
+3. Deploy a proxy contract to acess the data feed
+4. Read data from the dAPI
+
+![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-4.png)
```
Comment (albertov19) [inline builders/integrations/oracles/api3.md:157]: Transparent bg

**Review by albertov19 — CHANGES_REQUESTED** [review]
Some feedback

File: builders/integrations/oracles/api3.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+Requesting off-chain data essentially involves triggering an Airnode and getting its response through your smart contract. The smart contract in this case would be the requester contract, which will make a request to the desired off-chain Airnode and then capture its response.
+
+The requester calling an Airnode primarily focuses on two tasks:
+
+- Make the request
+- Accept and decode the response
+
+![API3 Airnode](/images/builders/integrations/oracles/api3/api3-2.png)
+
+**Here is an example of a basic requester contract to request data from an Airnode:**
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/integrations/oracles/api3.md:31]: oh yeah missed that one. thanks!

File: builders/integrations/oracles/api3.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+![API3 Moonbeam Diagram](/images/builders/integrations/oracles/api3/api3-banner.png)
+
+## Introduction {: #introduction } 
+
+Developers can use [Airnode](https://docs.api3.org/explore/airnode/what-is-airnode.html){target=_blank} to request off-chain data inside their smart contracts on Moonbeam networks. An Airnode is a first-party oracle that pushes off-chain API data to your on-chain contract. Airnode lets API providers easily run their own first-party oracle nodes. That way, they can provide data to any on-chain dApp that's interested in their services, all without an intermediary.
+
+An on-chain smart contract makes a request in the [RRP (Request Response Protocol)](https://docs.api3.org/reference/airnode/latest/concepts/){target=_blank} contract ([`AirnodeRrpV0.sol`](https://github.com/api3dao/airnode/blob/v0.11/packages/airnode-protocol/contracts/rrp/AirnodeRrpV0.sol){target=_blank}) that adds the request to the event logs. The Airnode then accesses the event logs, fetches the API data and performs a callback to the requester with the requested data.
+
+![API3 Airnode](/images/builders/integrations/oracles/api3/api3-1.png)
```
[the remark this answers is not in the document]
Comment (vanshwassan) [reply builders/integrations/oracles/api3.md:18]: Sure, here are all the images with a transparent bg:

api3-1
api3-2
api3-3
api3-4
api3-banner

[the remark this answers is not in the document]
Comment (eshaben) [reply builders/integrations/oracles/api3.md:18]: @vanshwassan for some reason, the grey lines in the second image aren't coming in clear like they are in the other images. can it be easily fixed?

[the remark this answers is not in the document]
Comment (vanshwassan) [reply builders/integrations/oracles/api3.md:18]: This should be better 👍🏻 
api3-2

[the remark this answers is not in the document]
Comment (eshaben) [reply builders/integrations/oracles/api3.md:18]: sweeet! thanks!

**Pull request merged**